scalatest/src/main/scala/org/scalatest/events/Event.scala

```diff
@@ -250,8 +250,10 @@ sealed abstract class Event extends Ordered[Event] with Product with Serializabl
       }
     }
 
-    def stackTrace(st: StackTraceElement): String =
+    def stackTrace(_st: StackTraceElement | Null): String = {
+      val st = _st.nn
```

Comment:
**Overridden java arguments are never null**
Argument should never be null, immediately remove nullness of parameter.